examples/49-hextile/hextile.cpp

```diff
@@ -63,21 +63,10 @@ namespace
 	public:
 		ExampleHextile(const char* _name, const char* _description, const char* _url)
 			: entry::AppI(_name, _description, _url)
-			, m_debug(BGFX_DEBUG_NONE)
-			, m_height(0)
 			, m_width(0)
-			, m_hexTileData()
-			, m_hextileProgram()
```

Comment:
is there a reason to not do default initialization?

Reply:
Those are automatically called.